tox.ini

```diff
@@ -139,6 +141,9 @@ deps =
     sanic-0.8: sanic>=0.8,<0.9
     sanic-18: sanic>=18.0,<19.0
     sanic-19: sanic>=19.0,<20.0
+    sanic-20: sanic>=20.0,<21.0
+    sanic-21: sanic>=21.0
```

Comment:
This might break CI with sanic 22, is that fine?

Reply:
oops